sirepo/package_data/static/js/raydata.js

```diff
@@ -531,6 +534,9 @@ SIREPO.app.directive('scansTable', function() {
                 $scope.reverseSortScans = ! $scope.reverseSortScans;
             };
 
+            if ($scope.analysisStatus === 'history') {
```

Comment:
I'd rather wait for a user to ask for this feature. We are only displaying 5 of the most recent scans so as long as they are in some sorted order (they are) the user should be able to look at all of them and grasp what is going on. If they want to change the sort order then they can click on one of the column headers and change the sort.

Reply:
71d7b187ccf540a8b22a16f86cc3ece57952d92d